src/slic3r/GUI/CalibrationPressureAdvDialog.cpp

```diff
@@ -1,875 +1,898 @@
+
+    DynamicPrintConfig full_print_config;
+    full_print_config.apply(*print_config);
+    full_print_config.apply(*printer_config);
+    full_print_config.apply(*filament_config);
+    
+    // --- scale ---
+    //models is created for nozzles from 0.1-2mm walls should be nozzle_size*4 spaced, scale xy model by widths down is futher
+
+    GCodeFlavor flavor = printer_config->option<ConfigOptionEnum<GCodeFlavor>>("gcode_flavor")->value;
+    const ConfigOptionFloats* nozzle_diameter_config = printer_config->option<ConfigOptionFloats>("nozzle_diameter");
+    assert(nozzle_diameter_config->size() > 0);
+    double nozzle_diameter = nozzle_diameter_config->get_at(0);//get extruderID too?
+
+    double first_layer_height = full_print_config.get_computed_value("first_layer_height");
+    double base_layer_height = full_print_config.get_computed_value("layer_height");
+    double er_width = print_config->get_abs_value("solid_infill_extrusion_width", nozzle_diameter);
+    double er_accel = full_print_config.get_computed_value("solid_infill_acceleration");
+    double er_speed = full_print_config.get_computed_value("solid_infill_speed");
+    double er_spacing = print_config->get_abs_value("external_perimeter_extrusion_spacing",1.0);
+
+    double default_er_width = print_config->get_abs_value("extrusion_width", nozzle_diameter);
+    double default_er_speed = full_print_config.get_computed_value("default_speed");
+    double default_er_accel = full_print_config.get_computed_value("default_acceleration");
+    double default_er_spacing = print_config->get_abs_value("extrusion_spacing", nozzle_diameter);
+    double spacing_ratio = full_print_config.get_computed_value("perimeter_overlap");
+    double spacing_ratio_external = full_print_config.get_computed_value("external_perimeter_overlap");
+    double filament_max_overlap = filament_config->get_computed_value("filament_max_overlap",0);//maybe check for extruderID ?
+
+    // --- translate ---
+    //bool autocenter = gui_app->app_config->get("autocenter") == "1";
+    bool has_to_arrange = plat->config()->opt_float("init_z_rotate") != 0;
+    has_to_arrange = true;
+    
+    /*if (!autocenter) {
+        const ConfigOptionPoints* bed_shape = printer_config->option<ConfigOptionPoints>("bed_shape");
+        Vec2d bed_size = BoundingBoxf(bed_shape->values).size();
+        Vec2d bed_min = BoundingBoxf(bed_shape->values).min;
+        model.objects[objs_idx[0]]->translate({ bed_min.x() + bed_size.x() / 2, bed_min.y() + bed_size.y() / 2, 5 * xyzScale - 5 });
+    }*/
+    
+    std::vector < std::vector<ModelObject*>> pressure_tower;
+    bool smooth_time = false;
+
+    std::string nozzle_diameter_str = std::to_string(nozzle_diameter);
+    std::replace(nozzle_diameter_str.begin(), nozzle_diameter_str.end(), ',', '.');
+    nozzle_diameter_str.erase(nozzle_diameter_str.find_last_not_of('0') + 2, std::string::npos);
+    std::cout << "Nozzle: " << nozzle_diameter_str << std::endl;

```

Comment:
did you forget to remove the console debugging ?

Reply:
yep :)